raspberryPI3/ros2_ws/src/detection/detection/obstacles_detection.py

```diff
@@ -0,0 +1,44 @@
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import Bool
+from interfaces.msg import Ultrasonic
+
+class ObstaclesDetection(Node):
+    
```

Comment:
You can add Docstring (IA does this very well), short example:

```py
class ObstaclesDetection(Node):
    """
    Node to detect obstacles using ultrasonic sensors.
    Publishes a Bool message to 'obstacles_detection' topic when an obstacle is detected.
    """
    ```

Reply:
Done !